content/docs/neon-auth/sdk/nextjs/objects/stack-server-app.md

```diff
@@ -0,0 +1,281 @@
+  <br />
+  <span><span style={{ display: "inline-block", width: "2ch" }}></span><a href="#stackserverappgetteamid" style={{ color: "#f59e42", textDecoration: "none" }}>getTeam</a><span style={{ color: "#60a5fa" }}>(id)</span><span style={{ color: "#60a5fa" }}>: </span><span style={{ color: "#facc15" }}>Promise</span><span style={{ color: "#60a5fa" }}>&lt;</span><span style={{ color: "#facc15" }}>ServerTeam|null</span><span style={{ color: "#60a5fa" }}>&gt;;</span></span><br />
+  <span><span style={{ display: "inline-block", width: "4ch" }}></span><span style={{ color: "#fff" }}>⤷ </span><a href="#stackserverappuseteamid" style={{ color: "#f59e42", textDecoration: "none" }}>useTeam</a><span style={{ color: "#60a5fa" }}>(id)</span><span style={{ color: "#60a5fa" }}>: </span><span style={{ color: "#facc15" }}>ServerTeam</span><span style={{ color: "#60a5fa" }}>;</span></span><br />
+  <span><span style={{ display: "inline-block", width: "2ch" }}></span><a href="#stackserverapplistteams" style={{ color: "#f59e42", textDecoration: "none" }}>listTeams</a><span style={{ color: "#60a5fa" }}>()</span><span style={{ color: "#60a5fa" }}>: </span><span style={{ color: "#facc15" }}>Promise</span><span style={{ color: "#60a5fa" }}>&lt;</span><span style={{ color: "#facc15" }}>ServerTeam[]</span><span style={{ color: "#60a5fa" }}>&gt;;</span></span><br />
+  <span><span style={{ display: "inline-block", width: "4ch" }}></span><span style={{ color: "#fff" }}>⤷ </span><a href="#stackserverappuseteams" style={{ color: "#f59e42", textDecoration: "none" }}>useTeams</a><span style={{ color: "#60a5fa" }}>()</span><span style={{ color: "#60a5fa" }}>: </span><span style={{ color: "#facc15" }}>ServerTeam[]</span><span style={{ color: "#60a5fa" }}>;</span></span><br />
+  <span><span style={{ display: "inline-block", width: "2ch" }}></span><a href="#stackserverappcreateteamoptions" style={{ color: "#f59e42", textDecoration: "none" }}>createTeam</a><span style={{ color: "#60a5fa" }}>([options])</span><span style={{ color: "#60a5fa" }}>: </span><span style={{ color: "#facc15" }}>Promise</span><span style={{ color: "#60a5fa" }}>&lt;</span><span style={{ color: "#facc15" }}>ServerTeam</span><span style={{ color: "#60a5fa" }}>&gt;;</span></span><br />
+  <span><span style={{ color: "#60a5fa" }}>{"};"}</span></span>
+</div>
+
+## Constructor
+
+Creates a new `StackServerApp` instance.
+
+### Parameters
+
+- `tokenStore`: The token store to use. Can be one of:
+  - `"nextjs-cookie"`: Uses Next.js cookies (recommended for Next.js apps)
+  - `"cookie"`: Uses browser cookies
+  - `{ accessToken: string, refreshToken: string }`: Uses provided tokens
+  - `Request`: Uses the provided request object
+- `secretServerKey`: The secret server key for your app.
+- `baseUrl`, `projectId`, `publishableClientKey`, `urls`, `noAutomaticPrefetch`: (see [StackClientApp](./stack-client-app.md) for details)
```

Comment:
Broken link, I think `./stack-client-app.md` wants to be `./stack-app` although another option is ti rename `stack-app.md` to `stack-client-app.md` and then use the expected `./stack-client-app`